src/repositories/friendship.repository.ts

```diff
@@ -66,4 +70,23 @@ export class FriendshipRepository extends Repository {
     });
     return body.friendship_status;
   }
+
+  async addAndRemoveBesties(addList?: string[] | number[], removeList?: string[] | number[]) {
+    const { body } = await this.client.request.send<FriendshipBestiesRepositoryResponseRootObject>({
+      url: `/api/v1/friendships/set_besties/`,
+      method: 'POST',
+      form: this.client.request.sign({
+        _csrftoken: this.client.state.cookieCsrfToken,
+        _uid: this.client.state.cookieUserId,
+        device_id: this.client.state.deviceId,
+        _uuid: this.client.state.uuid,
+        module: 'favorites_home_list',
+        source: 'audience_manager',
+        add: addList,
+        remove: removeList,
```

Comment:
What if these are undefined? Will the request fail? If so, add defaults.

Reply:
It wasn't handled. I also added a new error when the input isn't an array (because it would throw a 500)